### PR TITLE
add AbstractOrder.return_order

### PR DESCRIPTION
**Pull request by radekholy24**
Symmetrically to `AbstractOrder.complete_order`, we want to return an order *and* reduce the account.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,10 @@
 django-plans changelog
 ======================
 
+1.1.0 (unreleased)
+------------------
+* Add `AbstractOrder.return_order`
```
Comment (PetrDlouhy) [inline CHANGELOG:6]: `AbstractOrder.return_order()` to be clear that it is a function.

File: docs/source/index.rst [hunk 2]
```diff
@@ -47,6 +47,7 @@ Contributors:
 * Victor Safronovich <vsafronovich@gmail.com>
 * Dominik Kozaczko <http://dominik.kozaczko.info>
 * Petr Dlouhý <petr.dlouhy@email.cz>
+* Blender Kit <info@blenderkit.com>
```
Comment (PetrDlouhy) [inline docs/source/index.rst:50]: `BlenderKit <admin@blenderkit.com>` and/or give yourself a credit

File: demo/example/wsgi.py [hunk 3]
```diff
@@ -13,6 +13,7 @@
 framework.
 
 """
+
```
Comment (PetrDlouhy) [inline demo/example/wsgi.py:16]: Probably unnecessary modification of this file.

**Review by PetrDlouhy — CHANGES_REQUESTED** [review]
Please, only correct those few changes.

File: CHANGELOG (hunk 1 above)
Comment (radekholy24) [reply CHANGELOG]: Changed

File: docs/source/index.rst (hunk 2 above)
Comment (radekholy24) [reply docs/source/index.rst]: Changed

File: demo/example/wsgi.py (hunk 3 above)
Comment (PetrDlouhy) [reply demo/example/wsgi.py:16]: OK

**Review by PetrDlouhy — APPROVED** [review]

**Pull request merged**